src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -319,4 +358,52 @@ private void downloadJavaFXZip(String osarch, Path substratePath ) throws IOExce
         Logger.logDebug("Process zip javafx done");
     }
 
-}
+    private void downloadAndroidSdkZip() throws IOException {
+        Path sdk = configuration.getAndroidSdkPath();
+        String hostOs = configuration.getHostTriplet().getOs();
+        String androidSdkUrl = Strings.substitute(ANDROID_SDK_URL, Map.of("host", hostOs));
+        System.out.println("Downloading Android SDK...");
+        FileOps.downloadAndUnzip(androidSdkUrl, sdk.getParent(), "android-sdk.zip", sdk.getFileName().toString(), "");
+        System.out.println("Done");
+    }
+
+    private void downloadAdditionalAndroidLibs() throws IOException {
+        Path sdk = configuration.getAndroidSdkPath();
+        Path libsLocation = sdk.resolve("tools").resolve("lib").resolve("java11");
+
+        Files.createDirectories(libsLocation);
+        System.out.println("Downloading additional libs ...");
+        for (String url : ANDROID_DEPS) {
+            URL link = new URL(url);
+            String filename = url.substring(url.lastIndexOf('/')+1, url.length());
+            FileOps.downloadFile(link, libsLocation.resolve(filename));
+        }
+        System.out.println("Done");
+    }
+
+    private void androidSdkManager(String[] args) {
+        Path sdk = configuration.getAndroidSdkPath();
+        Path tools = sdk.resolve("tools");
+        Path libs = tools.resolve("lib");
+        Path additionalLibs = libs.resolve("java11");
+
+        ProcessRunner sdkmanager = new ProcessRunner("java", "-Dcom.android.sdklib.toolsdir=" + tools, "-classpath",
```

Comment:
What Java version is required? Probably better to use full path (i.e `$JAVA_HOME/bin/java`), verifying that JAVA_HOME is set, or even `publicConfig.getGraalPath()` if that works as well?